### DISCUSSION TITLE
How to handle software without usable official URLs - a whitlist for download portals?

### DISCUSSION THREAD

**Discussion by borque** [microsoft/winget-pkgs#97158] [General]
Hello!
Some projects offer their installers with non-static URLs (I thin "vanity" is the term used here?!) or simply don't want their bandwith to be used by third party installation tools. 
Some of these are frequently requested here (e.g. FileZilla) and don't have their own updaters so managing them using winget would be a perfect solution.
Is there any chance to create a kind of winget-whitelist of some trustworthy large download portals that often mirror this software and don't suffer from bandwith limitations or other restrictions and thus could be used by winget?
Perhaps combined with another list of manifests (to be selected and agreed upon by the staff) that can use URLs from that list?

**Comment by kairin** [comment]
> Hello!
> 
> Some projects offer their installers with non-static URLs (I thin "vanity" is the term used here?!) or simply don't want their bandwith to be used by third party installation tools. 
> 
> Some of these are frequently requested here (e.g. FileZilla) and don't have their own updaters so managing them using winget would be a perfect solution.
> 
> Is there any chance to create a kind of winget-whitelist of some trustworthy large download portals that often mirror this software and don't suffer from bandwith limitations or other restrictions and thus could be used by winget?
> 
> Perhaps combined with another list of manifests (to be selected and agreed upon by the staff) that can use URLs from that list?

thanks for bringing this up for attention.

as i'm practically new i just felt it was a bummer that i could not contribute based on my interpretation (which turns out to be incorrect) will try others and learn along the way.

**Comment by denelon** [comment]
WinGet doesn't host any binaries and isn't designed to do that. 

If a publisher has an "official" relationship with a mirror (something we can verify via the official "site"), and that mirror doesn't mind traffic that isn't going to view "adds" or otherwise have a mechanism to monetize the cost of hosting, this may be an option.

**Comment by rwiesbach** [comment]
Half a year later now, has there been any decision on that issue?
